scheduler.py

```diff
@@ -47,17 +47,15 @@ def convert_to_timeform(time):
     def find_time(n):
         """find first instance of window of empty values in dict"""
         free_in_a_row = 0
-        for index, date in enumerate(schedule):
-            
+        for index, date in enumerate(schedule):     
             if (np.isnan(schedule[date])):
                 free_in_a_row += 1
             else:
                 free_in_a_row = 0
-            if (free_in_a_row >= n):
-                return index-n
-            index += 1
+            if (free_in_a_row == n):
+                return index-n+1
```

Comment:
## tl;dr in advance:

1) PR approved, great work.
2) If you contribute more, force pull first to avoid errors: tests to be written for future feature adds.
3) Follow the new CONTRIBUTING.md guidelines (shouldn't be much different from what you're already doing).

## Regarding this PR: Approved!

Thank you for your continued contributions! You are a great help and embody the spirit of open source. I can tell this might have taken a bit of thought (since mentally checking behavior of the function certainly took some time). I sincerely appreciate the time you put into this.

Both of your additions are completely fair and are much closer, both in letter and in spirit, to the expected behavior of the "find_time()" function. I will approve this PR now! Great work.

## P.S.

So that you are aware, here are my current plans for after I approve this PR: the wip-2-develop-tests branch currently has a changed project structure (using the src/ and tests/ convention). I am going to force pull your changes into that branch, re-add the modifications, and then merge with the main branch in order to add testing to the project through pytest and coverage which will improve workflow, code cleanliness, and time efficiency. I will also change the CONTRIBUTING.md file with more concrete information on how we have been operating so far. These changes aren't in line with the precedent I set before - I apologize for that and for any inconvenience that this might raise, but rest assured, that won't affect this PR at all. It will only matter if you continue to contribute.

If you go on to contribute more, I would be extremely happy to have you! With what I just said in mind, to proceed onto other contributions, you will probably have to force pull changes from the main branch post-merge into your local repo to avoid diverging errors. For any future feature adds, I will also ask for associated tests to be written in the tests/ folder: this is actually to the benefit of the developer, i.e. you, so that the anticipated behavior is written in advance and errors are caught earlier so that commits happen faster and with more intentionality. As usual, follow the CONTRIBUTING.md guidelines.